app/containers/markdown/previewFormatText.ts

```diff
@@ -11,6 +11,7 @@ export const previewFormatText = (msg: string) => {
 	// Removes sequential empty spaces before to use removeMarkdown,
 	// because with some edge cases the library takes a long time to finish the process
 	m = m.replace(/\s+/g, ' ');
+	m = m.replace(/\*\s+|\s+\*/g, '*')
```

Comment:
Can you fix it inside removeMarkdown lib and open a PR there?
While we do other regex here, those are only relevant for our use case.
Removing emphasis is something that the lib should be taking care of.

Reply:
Can i create a patch because i they have a test case which doesn't allow a space between character and emphasis characters
https://github.com/zuchka/remove-markdown/blob/main/test/remove-markdown.js#L72